### DISCUSSION TITLE
Drawing a simple line in the visual editor

### DISCUSSION THREAD

**Discussion by jcolafrancesco** [winder/Universal-G-Code-Sender#2618] [Help]
Hi,
Is there any way to draw a simple line from within the editor?
Could be very useful to cut a piece in two parts.
I tried a rectangle with 0 width and "on path" cut, without success (seems like 0 value for rectangle dimensions is not allowed).

Best,

**Comment by breiler** [comment]
No that is currently not possible yet. I have wanted to implement this, but it requires a whole new type of path editor which is quite difficult to create.

**Reply by jcolafrancesco** [reply]
Would you mind describing what makes the path editor unadapted today?

Is it that it assume closed path?

Is there a way to make the 0 width  on path rectangle trick to work?

(I wonder if we cannot find a kind of quick and dirty workaround).

Not a Java expert but depending on your answer I may give it a try.
